MovingBlocks/Terasology
# added extra menu in pause screen

### Contains
Added an **Extra menu** in  **pause menu** which includes **"Metrics Menu "**, **"Issue Reporter"**, **"Developer Tools"** along with **"Back"** Button #3608


### How to test
Run game and press esc key.
You will be able to see a new button **"Extra"** which includes **"Metrics Menu "**, **"Issue Reporter"**, **"Developer Tools"** along with **"Back"** Button.

### Outstanding before merging


- [x] Still have to adjust the text on button which is having **'?'** on left and right side of text
![52303328-b7bdcf00-29b5-11e9-83d1-dd330f104f71](https://user-images.githubusercontent.com/22188087/52348034-6101d500-2a49-11e9-8fd7-bcda96004850.png)
